### PR TITLE
Update README.md file (#1656)

### PR DESCRIPTION
**Pull request by nbougalis**
* Update links, so as not to reference files that are no longer available on the ripple.com website. 
* Link to the Knowledge Center instead of the Ripple wiki.

Reviews: @mDuo13, @yana-novikova

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,15 +102,13 @@ README for more details.
...
 
 ###For more information:
-* Ripple Wiki - https://ripple.com/wiki/
-* Ripple Primer - https://ripple.com/ripple_primer.pdf
-* Ripple Primer (Market Making) - https://ripple.com/ripple-mm.pdf
-* Ripple Gateway Primer - https://ripple.com/ripple-gateways.pdf
-* Consensus - https://wiki.ripple.com/Consensus
+* [Ripple Knowledge Center](https://ripple.com/learn/)
+* [Ripple Executive Summary for Financial Institutions](https://ripple.com/solutions/executive-summary-for-financial-institutions/)
+* [Ripple Solutions](https://ripple.com/files/ripple_solutions_guide.pdf)
```
Comment (mDuo13) [inline README.md]: Add a link to the [Ripple Developer Center](https://ripple.com/build/)

**Comment by mDuo13** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (nbougalis) [inline README.md]: Thanks - wilco.

**Comment by yana-novikova** [comment]
I was thinking to change README to include more info about XRP and update the visuals. I can put a doc together combining/summarizing info from from this pages:
https://ripple.com/xrp-portal/
https://ripple.com/files/ripple_vision.pdf

Let me know if this sounds good.

**Comment by nbougalis** [comment]
Closed pending update.

**Pull request closed without merging**